src/webgpu/shader/validation/extension/dual_source_blending.spec.ts

```diff
@@ -0,0 +1,38 @@
+export const description = `
+Validation tests for the dual_source_blending extension
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+g.test('use_blend_src_requires_extension_enabled')
+  .desc(
+    `Checks that the blend_src attribute is only allowed with the extension dual_source_blending
+     supported`
+  )
+  .beforeAllSubcases(t => {
+    t.selectDeviceOrSkipTestCase({ requiredFeatures: ['dual-source-blending'] });
+  })
+  .params(u => u.combine('enableExtension', [true, false] as const))
```

Comment:
I thought we should also test on feature-not-supported devices that not using the feature ends up in validation error.

``` suggestion
  .params(u => u.combine('enableExtension', [true, false] as const))
  .beforeAllSubcases(t => {
    if (t.params.enableExtension) {
      t.selectDeviceOrSkipTestCase({ requiredFeatures: ['dual-source-blending'] });
    }
  })
```

Reply:
I've updated the test here. PTAL, thanks!